### PR TITLE
Add pagination

### PR DESCRIPTION
**Pull request by Daniil9596**
All done: 
1) Added pagination where it need.
2) Replaced useless code.
3) Fixed tests for pagination.

### CODE REVIEW

File: src/test/java/com/epam/lab/paymentsystem/service/CardServiceTest.java [hunk 1]
```diff
@@ -59,10 +68,10 @@ public void startUp() {
     }
 
     @Test
-    public void testGetAllCardsByAccountIdReturnsCardsList() {
+    public void testGetAllCardsByAccountIdReturnsCardsPage() {
         when(accountService.getAccountById(accountId)).thenReturn(account);
-        when(cardRepository.getAllByAccount(account)).thenReturn(cards);
-        assertEquals(cards, cardService.getAllCardsByAccountId(accountId));
+        when(cardRepository.getAllByAccount(account, pageable)).thenReturn(cardsPage);
+        assertEquals(cardsPage, cardService.getAllCardsByAccountId(accountId, pageable));
```
Comment (Meisssshu) [inline src/test/java/com/epam/lab/paymentsystem/service/CardServiceTest.java]: Please, add message and reformat code

File: src/test/java/com/epam/lab/paymentsystem/service/OperationServiceTest.java [hunk 2]
```diff
@@ -158,12 +97,12 @@ public void startUp() {
...
-  public void testGetAllOperationsReturnsOperationsList() {
+  public void testGetAllOperationsReturnsOperationsPage() {
     when(userService.getCurrentUserLogin()).thenReturn(user.getLogin());
     when(cardService.getAllCardsByLogin(user.getLogin())).thenReturn(cards);
-    when(operationRepository.getAllBySourceCardIsInOrTargetCardIsIn(cards, cards))
-        .thenReturn(operations);
-    assertEquals(operations, operationService.getAllOperations());
+    when(operationRepository.getAllBySourceCardIsInOrTargetCardIsIn(cards, cards, pageable))
+        .thenReturn(operationsPage);
+    assertEquals(operationsPage, operationService.getAllOperations(pageable));
```
Comment (Meisssshu) [inline src/test/java/com/epam/lab/paymentsystem/service/OperationServiceTest.java]: Please, add message and reformat code

File: src/test/java/com/epam/lab/paymentsystem/service/OperationServiceTest.java [hunk 3]
```diff
@@ -191,15 +130,17 @@ public void testMakePayment() {
...
   @Test
-  public void testGetAllOperationsByAccountReturnsOperationsList() {
+  public void testGetAllOperationsByAccountReturnsOperationsPage() {
     when(cardService.getAllCardsByAccountId(accountId)).thenReturn(cards);
-    when(operationRepository.getAllBySourceCardIsInOrTargetCardIsIn(cards, cards))
-        .thenReturn(operations);
-    assertEquals(operations, operationService.getAllOperationsByAccount(accountId));
+    when(operationRepository.getAllBySourceCardIsInOrTargetCardIsIn(cards, cards, pageable))
+        .thenReturn(operationsPage);
+    assertEquals(operationsPage, operationService.getAllOperationsByAccount(accountId, pageable));
```
Comment (Meisssshu) [inline src/test/java/com/epam/lab/paymentsystem/service/OperationServiceTest.java]: Please, add message and reformat code

File: src/test/java/com/epam/lab/paymentsystem/service/OperationServiceTest.java [hunk 4]
```diff
@@ -191,15 +130,17 @@ public void testMakePayment() {
...
   }
 
   @Test
-  public void testGetAllOperationsByCardReturnsOperationsList() {
-    assertEquals(operations, operationService.getAllOperationsByCard(cardId));
+  public void testGetAllOperationsByCardReturnsOperationsPage() {
+    when(operationRepository.getAllBySourceCardIdOrTargetCardId(cardSrcId, cardSrcId, pageable))
+            .thenReturn(operationsPage);
+    assertEquals(operationsPage, operationService.getAllOperationsByCard(cardId, pageable));
```
Comment (Meisssshu) [inline src/test/java/com/epam/lab/paymentsystem/service/OperationServiceTest.java]: Please, add message and reformat code

Comment (Daniil9596) [reply src/test/java/com/epam/lab/paymentsystem/service/OperationServiceTest.java]: Fixed all above)

File: src/test/java/com/epam/lab/paymentsystem/service/CardServiceTest.java [hunk 5]
```diff
@@ -93,8 +96,7 @@ public void testCreateCardReturnsCard() {
   public void testCreateCardThrowsException() {
     when(accountService.getAccountById(accountId)).thenReturn(account);
     when(userService.getUserByLogin(user.getLogin())).thenReturn(null);
-    assertThrows(CardArgumentException.class,
-        () -> cardService.createCard(cardDto),
-        "No such user");
+    assertThrows(
+        CardArgumentException.class, () -> cardService.createCard(cardDto), "No such user");
```
Comment (Meisssshu) [inline src/test/java/com/epam/lab/paymentsystem/service/CardServiceTest.java]: Please, reformat code

Comment (Daniil9596) [reply src/test/java/com/epam/lab/paymentsystem/service/CardServiceTest.java]: Sorry, I forgot to fix this place) 
Fixed.

**Pull request merged**